cfgov/jinja2/v1/job-description-page/index.html

```diff
@@ -28,14 +23,18 @@ <h1>{{ page.title }}</h1>
                     {{ time.render(page.close_date, {'date': true}, text_format=True) }}
                 {% endif %}
             </dd>
-            <dt>Region:</dt>
-            <dd>{{ page.region }}</dd>
-            
-            {% if page.region and page.region.name in ["Midwest region", "Northeast region", "Southeast region", "West region" ]%}
-            <dt class="u-hidden">Location:</dt>
-            <dd class="u-hidden">
-                {{ regional_cities[page.region.name] }}
-            </dd>
+            {% if location_type == 'region' %}
+                <dt>Region:</dt>
+                <dd>{{ page.location }}<span class="u-hidden"> ({{states|join(', ')}})</span></dd>
+                {% if cities %}
+                    <dt class="u-hidden">Location:</dt>
+                    <dd class="u-hidden">
+                    {{ cities|join('; ') }}
+                    </dd>
+                {% endif  %}
+            {% else %}
+                <dt>Location:</dt>
+                <dd>{{ cities[0] if cities|length else page.location }} {{ ' or remote' if page.allow_remote else '' }}</dd>
```

Comment:
1. Is remote only allowed for jobs in cities, and not for jobs in regions? If so, it might be helpful to add some help text about this to the Wagtail edit page.
2. Minor thing, but there's a double space between the city name and "  or remote".

Reply:
@chosak That's a good point! The remote option is only available for jobs with office locations, so I added help text to that effect in https://github.com/cfpb/cfgov-refresh/pull/3748/commits/6caf75363c7dc013b321b0bb845db4452dae4a73 (and also updated the spacing around 'or remote')
<img width="557" alt="screen shot 2018-01-29 at 3 08 12 pm" src="https://user-images.githubusercontent.com/778171/35539406-3f63fa9a-0506-11e8-9b58-4343b6985e1b.png">